src/main/java/org/sonar/plugins/resharper/ReSharperSensor.java

```diff
@@ -170,7 +172,7 @@ private static void logSkippedIssueOutsideOfSonarQube(ReSharperIssue issue, File
   }
 
   private static void logSkippedIssue(ReSharperIssue issue, String reason) {
-    LOG.info("Skipping the ReSharper issue at line " + issue.reportLine() + " " + reason);
+    LOG.debug("Skipping the ReSharper issue at line " + issue.reportLine() + " " + reason);
```

Comment:
@rikvandenberg do you still get such messages? I can image that this would happen only if you have several projects with the same name in your solution?

it could also happen if you use exclusions